File: LightningConstructs/Source/Dictionary/Declarations.vb
    Public Partial Class Dictionary(Of TKey, TValue)
        Private BaseKeysArray As TKey() = New TKey(32767) {}
        Private BaseValuesArray As TValue() = New TValue(32767) {}
        Private BaseClaimedBitmap As Boolean() = New Boolean(32767) {}
        Private FreePointer As System.Int32 = 0
        Private ReadOnly ReclaimedPointers As System.Collections.Generic.Queue(Of System.Int32) = New System.Collections.Generic.Queue(Of System.Int32)()
        Private ReclaimedCount As System.Int32 = 0
    End Class

File: LightningConstructs/Source/Dictionary/Declarations.cs


namespace Lightning
{
    public partial class Dictionary<TKey, TValue>
    {
        private TKey[] BaseKeysArray = new TKey[32768];
        private TValue[] BaseValuesArray = new TValue[32768];
        private bool[] BaseClaimedBitmap = new bool[32768];
        private System.Int32 FreePointer = 0;
        private readonly System.Collections.Generic.Queue<System.Int32> ReclaimedPointers = new System.Collections.Generic.Queue<System.Int32>();
        private System.Int32 ReclaimedCount = 0;
    }
}